SFTPStream.md

```diff
@@ -7,32 +7,54 @@ SFTPStream events
 
 **Server-only events**
 
+In all cases, responses to the various events **must** include the request's `reqID` as
+their first parameter. The `sftpStream` that the event fired on is the same object 
+that the various response methods should be invoked using.
+
 * **OPEN**(< _integer_ >reqID, < _string_ >filename, < _integer_ >flags, < _ATTRS_>attrs)
+ - respond with `sftpStream.handle()` with your newly-opened file handle. The file handle
+   should uniquely identify this opened file for this connection.
 
 * **READ**(< _integer_ >reqID, < _Buffer_ >handle, < _integer_ >offset, < _integer_ >length)
+ - respond with `sftpStream.data()` with the requested data. Respond with 
+   `sftpStream.status(ssh2.SFTP_STATUS_CODES.EOF)` if there is no more data to return.
 
 * **WRITE**(< _integer_ >reqID, < _Buffer_ >handle, < _integer_ >offset, < _Buffer_ >data)
 
 * **FSTAT**(< _integer_ >reqID, < _Buffer_ >handle)
-
+ - respond with `sftpStream.attrs()`
+ 
 * **FSETSTAT**(< _integer_ >reqID, < _Buffer_ >handle, < _ATTRS_ >attrs)
 
 * **CLOSE**(< _integer_ >reqID, < _Buffer_ >handle)
+ - respond with `sftpStream.status(ssh2.SFTP_STATUS_CODES.OK)` on success.
 
 * **OPENDIR**(< _integer_ >reqID, < _string_ >path)
+ - respond with `sftpStream.handle()` with the directory handle. That handle
+   should uniquely identify this opened directory for this client.
 
 * **READDIR**(< _integer_ >reqID, < _Buffer_ >handle)
+ - the `handle` that you responded to the `OPENDIR` event with is passed to you
+   as a second parameter to identify which directory is being read. Respond with
+   `sftpStream.name()` and an array of file entries. The event will then fire 
```

Comment:
You don't *have to* respond with `sftpStream.name()`. You could respond using `sftpStream.status()` if necessary.